src/main/java/bit/javaoop/Manager.java

```diff
@@ -0,0 +1,53 @@
+package bit.javaoop;
+
+public class Manager extends Employee{

```

Comment:
Mogłeś przemyśleć stworzenie Manager'a jako klasy abstrakcynej.

Klasa abstrakcyjna wprowadza obowiązek zaimplementowania swoich abstrakcyjnych metod wśród wszystkich swoich dzieci. 
+ Zauważ, że każdy Manager ma inną politykę zatrudniania pracowników. Chcemy zatem wymusić na nich by ją "po swojemu" zaimplementowali. Tworzymy więc abstrakcyjną metodę canHire(). Dzięki temu szczegółowi Managerzy dziedzicząc po niej muszą ją zaimplementować. Taka kontrola kodu.
+ Twoja obecna klasa Manager posiada metodę canHire. lecz każdy z szczegółowych Managerów posiada swoją metodkę do sprawdzania czy może zatrudnić pracownika. Oznacza to, że tworząc tablicę managerów o typie Manager nie możemy sprawdzić, czy któryś manager z tej listy może kogoś tak naprawdę zatrudnić. Tj. nie możemy zastosować tutaj pełni funkcjonalności polimorfizmu, a to chcieliśmy głównie osiągnąć w tym zadanku. Jedyne co możemy napisać to `managers[counter].canHire()` i da nam to informacje, czy mamy jeszcze miejsce w tablicy managera, a nie czy, jak sugeruje nazwa metody, rzeczywiście możemy kogoś zatrudnić.